src/components/field/Field.vue

```diff
@@ -15,6 +15,7 @@
         props: {
             type: String,
             label: String,
+            labelFor: String,
```

Comment:
I don't know if it's ok or add `v-bind="$attrs"` to `<label>` tag (like checkbox component). 
At the moment custom attribute go to wrapper `<div>` but i don't think that it helps in this tag

Reply:
Thanks @jtommy, I tried renaming this prop to `for` but since it's a reserved word (I think) the `        <label class="label" :for="for" v-if="label">{{ label }}</label>
` in the template refuses to build.  Should I leave it as `labelFor` or is there a way in Vue to escape the reserved word and use `for` anyway?

Reply:
@tmepple Yes, you are right! I don't like but i think it's the only solution.